includes/class-wc-api.php

```diff
@@ -19,6 +19,12 @@
 class WC_API {
 
 	/**
+	 * WP REST API namespace/version.
+	 * @todo to be removed when the wp-api branch is merged.
+	 */
+	const REST_API_NAMESPACE = 'wc/v1';
```

Comment:
Better remove it and use the follow variable inside each class:

```
	/**
	 * Endpoint namespace.
	 *
	 * @var string
	 */
	public $namespace = 'wc/v1';
```

It make easy in the future, so we can just extend or current classes and generate new versions.